src/components/encryption.ts

```diff
@@ -261,26 +268,48 @@ export class EncryptedEventBroker {
         this.onEphemeralEvent(presenceEv);
     }
 
+    private async onSync(userId: string, state: string, data: {nextSyncToken: string; catchingUp: boolean;}) {
+        log.debug(`${userId} sync is ${state}`);
+        if (state === "SYNCING" || state === "PREPARED" && data.nextSyncToken) {
```

Comment:
I assume that the `||` gets resolved first, and then `&&`. Could you add a parenthesis around one part, to explicitly show your intent?